S21/hvkone/ui_txt_conv.py

```diff
@@ -0,0 +1,81 @@
+#---------------------------------
```

Comment:
Overall it looks fine, your ideas are great. I would suggest adding more comments on your individual functions to help the reviewers understand them more.

Reply:
Will do! Thank you for the review.

Reply:
Your welcome and thank you for understanding my points.